src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1175,6 +1187,7 @@ void cbmc_parse_optionst::help()
     " --yices                      use Yices\n"
     " --z3                         use Z3\n"
     " --refine                     use refinement procedure (experimental)\n"
+    " --pass                       use pass procedure (experimental)\n"
```

Comment:
So ... what is the pass procedure? Would be nice to be user-friendly in the --help output.